### PR TITLE
Release 15.1.0

### PR DESCRIPTION
**Pull request by wbclark**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,13 +1,56 @@
...
+
+## [15.0.1](https://github.com/theforeman/puppet-foreman/tree/15.0.1) (2020-06-15)
+
+[Full Changelog](https://github.com/theforeman/puppet-foreman/compare/14.0.2...15.0.1)
+
+**Merged pull requests:**
+
+- Fixes [\#30026](https://projects.theforeman.org/issues/30026) - Ensure Foreman is provisioned before puppetdb [\#857](https://github.com/theforeman/puppet-foreman/pull/857) ([ehelms](https://github.com/ehelms))
+
+## [14.0.2](https://github.com/theforeman/puppet-foreman/tree/14.0.2) (2020-06-05)
```
Comment (ekohl) [inline CHANGELOG.md:33]: Normally I leave these since they're older than 15.0.0 which means they're only cherry picks.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,13 +1,56 @@
...
+
+- Fixes [\#30456](https://projects.theforeman.org/issues/30456) - Fix missing icons on /pub page [\#867](https://github.com/theforeman/puppet-foreman/pull/867) ([adamruzicka](https://github.com/adamruzicka))
+- fix: indent for rails\_cache\_store redis type [\#859](https://github.com/theforeman/puppet-foreman/pull/859) ([ministicraft](https://github.com/ministicraft))
+- Fixes [\#30026](https://projects.theforeman.org/issues/30026) - Ensure Foreman is provisioned before configuring puppetdb [\#851](https://github.com/theforeman/puppet-foreman/pull/851) ([ekohl](https://github.com/ekohl))
+
+## [15.0.2](https://github.com/theforeman/puppet-foreman/tree/15.0.2) (2020-08-03)
+
+[Full Changelog](https://github.com/theforeman/puppet-foreman/compare/15.0.1...15.0.2)
+
+## [15.0.1](https://github.com/theforeman/puppet-foreman/tree/15.0.1) (2020-06-15)
```
Comment (ekohl) [inline CHANGELOG.md:25]: This and 15.0.2 require some manual puzzling.

**Comment by ekohl** [comment]
> We should first merge #847, but that requires some work.

Thinking more about it, let's release this without merging it since it's a breaking change. Then we have a more recent 15.x with as much changes as possible and it's not needed for RC1.

**Comment by wbclark** [comment]
@ekohl I fixed some of the weird instances of PRs being listed under the incorrect release. Please take a look!

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,44 @@
...
+- Add foreman\_statistics plugin [\#855](https://github.com/theforeman/puppet-foreman/pull/855) ([ezr-ondrej](https://github.com/ezr-ondrej))
+- Fixes [\#29892](https://projects.theforeman.org/issues/29892) - Use server certs for websockets [\#846](https://github.com/theforeman/puppet-foreman/pull/846) ([ekohl](https://github.com/ekohl))
+- Switch to postgresql::postgresql\_password [\#845](https://github.com/theforeman/puppet-foreman/pull/845) ([mmoll](https://github.com/mmoll))
+
+**Fixed bugs:**
+
+- Fixes [\#30456](https://projects.theforeman.org/issues/30456) - Fix missing icons on /pub page [\#867](https://github.com/theforeman/puppet-foreman/pull/867) ([adamruzicka](https://github.com/adamruzicka))
+- fix: indent for rails\_cache\_store redis type [\#859](https://github.com/theforeman/puppet-foreman/pull/859) ([ministicraft](https://github.com/ministicraft))
+- Fixes [\#30026](https://projects.theforeman.org/issues/30026) - Ensure Foreman is provisioned before configuring puppetdb [\#851](https://github.com/theforeman/puppet-foreman/pull/851) ([ekohl](https://github.com/ekohl))
```
Comment (ekohl) [inline CHANGELOG.md:18]: You can leave this out since it was cherry picked to 15.0.1.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,44 @@
 # Changelog
 
+## [15.1.0](https://github.com/theforeman/puppet-foreman/tree/15.1.0) (2020-08-07)
+
+[Full Changelog](https://github.com/theforeman/puppet-foreman/compare/15.0.2...15.1.0)
+
+**Implemented enhancements:**
+
+- Fixes [\#30078](https://projects.theforeman.org/issues/30078) - add parameter to accept a hostgroup config hash [\#863](https://github.com/theforeman/puppet-foreman/pull/863) ([apatelKmd](https://github.com/apatelKmd))
+- Add foreman\_statistics plugin [\#855](https://github.com/theforeman/puppet-foreman/pull/855) ([ezr-ondrej](https://github.com/ezr-ondrej))
```
Comment (ekohl) [inline CHANGELOG.md:10]: You can leave this out since it was cherry picked to 15.0.2.

File: HISTORY.md [hunk 5]
```diff
@@ -1,3 +1,71 @@
+## [15.0.2](https://github.com/theforeman/puppet-foreman/tree/15.0.2) (2020-08-03)
```
Comment (ekohl) [inline HISTORY.md:1]: If you modify the latest version (15.1.0 now) I'd also add it to HISTORY.md immediately. You'll need to do that on whenever you do the next version and you can be kind to your future self.

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,44 @@
...
+- Fixes [\#29892](https://projects.theforeman.org/issues/29892) - Use server certs for websockets [\#846](https://github.com/theforeman/puppet-foreman/pull/846) ([ekohl](https://github.com/ekohl))
+- Switch to postgresql::postgresql\_password [\#845](https://github.com/theforeman/puppet-foreman/pull/845) ([mmoll](https://github.com/mmoll))
+
+**Fixed bugs:**
+
+- Fixes [\#30456](https://projects.theforeman.org/issues/30456) - Fix missing icons on /pub page [\#867](https://github.com/theforeman/puppet-foreman/pull/867) ([adamruzicka](https://github.com/adamruzicka))
```
Comment (ekohl) [inline CHANGELOG.md:16]: You can leave this out since it was cherry picked to 15.0.2.

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,44 @@
...
+- fix: indent for rails\_cache\_store redis type [\#859](https://github.com/theforeman/puppet-foreman/pull/859) ([ministicraft](https://github.com/ministicraft))
```
Comment (ekohl) [inline CHANGELOG.md:17]: You can leave this out since it was cherry picked to 15.0.2.

**Comment by wbclark** [comment]
@ekohl Updated

**Pull request merged**